contrib/pg_tde/documentation/CONTRIBUTING.md

```diff
@@ -27,8 +107,7 @@ The steps are the following:
 2. Clone the repository on your machine:
 
 ```sh
-git clone git@github.com:<your_name>/pg_tde.git
-```
+git clone git@github.com:<your-name>/pg_tde.git
```

Comment:
outdated link, now its the postgres repository. Also needs `--recursive`.

Reply:
Could you please type the correct command?